__tests__/endpoints.test.js

```diff
@@ -129,6 +129,7 @@ describe("/api/articles/:article_id", () => {
             topic: expect.any(String),
             created_at: expect.any(String),
             votes: expect.any(Number),
+            comment_count: expect.any(String),
```

Comment:
Ideally, you shouldn't be amending passing tests. .toMatchObject should still pass if a new property is added, so better to split this into its own new test for comment_count specifically.